packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -22,20 +22,21 @@
   "scripts": {
     "start": "backstage-cli package start",
     "build": "backstage-cli package build",
+    "export-dynamic": "janus-cli package export-dynamic-plugin",
     "lint": "backstage-cli package lint",
     "test": "backstage-cli package test",
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack"
```

Comment:
we will need the preversion script here, as in https://github.com/janus-idp/backstage-plugins/blob/main/plugins/3scale-backend/package.json#L21, if we want the `semantic-release` to work correctly.